server/static/sass/4-elements/_all.scss

```diff
@@ -23,43 +26,52 @@ h3,
 h4,
 h5,
 h6,
-p {
+p,
+figure,
+blockquote {
   margin: 0;
   padding: 0;
 }
 
-// Set base typography proportions for headers, paragraphs, etc.
-// sizing variables in settings/_breakpoints.scss
-h1 {
```

Comment:
Pretty sure these rules are being covered elsewhere but wanted to make sure you checked to see if these updates effected any of the other donor app pages.

Reply:
There are some mild regressions on the `/donate` page, specifically around the `h1` size. I'll address shortly.